kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -544,10 +544,30 @@
           return 346;
         }
       },
-      gridOffset() {
-        return this.isRtl
-          ? { marginRight: `${this.sidePanelWidth + 24}px` }
-          : { marginLeft: `${this.sidePanelWidth + 24}px` };
+      gridStyle() {
+        let style = {};
```

Comment:
I think once this is pushed into develop, this will cause a linting error because of the addition of the prefer const linting rule.
```suggestion
        const style = {};
```

Reply:
This was an error, I have removed it.

Reply:
Ah yeah, no worries, using two syntaxes (first time rewriting the object, and the second time only adding a new attribute) for the very same thing didn't help much with this